test/draw/gpu/ToRasterTest.ooc

```diff
@@ -0,0 +1,27 @@
+use geometry
+use unit
+use draw
+use draw-gpu
+use base
+use opengl
+
+ToRasterTest: class extends Fixture {
+	toRasterTestFunction: func (sourceImage: RasterImage) {
+		gpuImage := context createImage(sourceImage)
+		raster := gpuImage toRaster()
+		expect(raster distance(sourceImage), is equal to(0.0f))
+		sourceImage free()
+		gpuImage free()
+		raster free()
+	}
+	init: func {
+		super("ToRaster")
+		this add("toRaster bgra", || this toRasterTestFunction(RasterBgra open("test/draw/gpu/input/Flower.png")))
```

Comment:
This is a new way of writing tests, to be sure. I won't object to it, though, as it looks pretty nice.